model/tasks/SendLtiOutcomeTask.php

```diff
@@ -118,12 +119,12 @@ private function sendLtiOutcome(taoResultServer_models_classes_OutcomeVariable $
         return true;
     }
 
-    private function buildXMLMessage($sourcedId, $grade)
+    private function buildXMLMessage($sourcedId, $grade): string
     {
         $language = 'en-us';
         $operation = 'replaceResultRequest';
         $body = '<?xml version = "1.0" encoding = "UTF-8"?>
-                <imsx_POXEnvelopeRequest xmlns = "http://www.imsglobal.org/lis/oms1p0/pox">
```

Comment:
not in the scope of the task but this looks bad, 
maybe in the near future we can use a tpl file from where were are getting this information so we don't need several changes when changing just a part of the xml

Reply:
agree